Purchases/Support/BeginRefundRequestHelper.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  BeginRefundRequestHelper.swift
```

Comment:
how would you feel about `BeginRefundRequestOrchestrator`, to follow an existing naming pattern?

Reply:
I'd usually use API for a Facade, and Orchestrator for classes that orchestrate the flow of logic between different agent classes. 
This case would fall closer to the Facade IMO since it mostly wraps the SK2 calls. `BeginRefundRequestWrapper` would also work for the same reason

Reply:
Something that might inform on naming -- we might eventually also want this class to distinguish between mac and ios to support the mac version of this method

Reply:
in a sense, this is an orchestrator because it orchestrates the flow between sk2 and a potential other store. though i suppose that's not likely going to happen.